sdk/nodejs/appservice/zMixins.ts

```diff
@@ -88,7 +89,9 @@ export type CallbackFunctionAppArgs<C extends Context<R>, E, R extends Result> =
      * the Azure will call into each time the FunctionApp it is is invoked.
      */
     callbackFactory?: CallbackFactory<C, E, R>;
+}
 
+export type CallbackFunctionAppArgs<C extends Context<R>, E, R extends Result> = CallbackArgs<C, E, R> & util.Overwrite<FunctionAppArgs, {

```

Comment:
note: since we're making this cahnge, we probably should just exract out what we need from FunctionAppArgs and place directly in here.  it will fit the awsx approach.